cnv/cnv-configuring-pxe-booting.adoc

```diff
@@ -4,6 +4,19 @@ include::modules/cnv_document_attributes.adoc[]
 :context: pxe-booting
 toc::[]
 
+PXE booting, or network booting, is available in {ProductName}.
+Network booting allows a computer to boot and load an
+operating system or other program without requiring a locally attached
+storage device. For example, you can use it to choose your desired OS
+image from a PXE server when deploying a new host.
+
+.Prerequisites
+
+//note: the xref below needs to be changed after the CNV buckets are set up.
+
+* A Linux bridge must be connected, as described xref:../../cnv/cnv-attaching-vm-multiple-networks.adoc#connecting-resource-bridge-network-cnv-attaching-multiple-networks[here].
```

Comment:
We avoid "here" and "click here" statements. I would just make the word "connected" the link. The user will see it's a link and intuit that they can click there for more information.